### PR TITLE
Give mobile users the option to export the calendar

### PR DESCRIPTION
**Pull request by bp323**
Mobile views where never part of the specification. However, we should present mobile users with the option to export their personal calendar to the phone and provide a message that tells them to use Timetable on a tablet or computer if they wish to make edits to it.

### CODE REVIEW

**Comment by bp323** [comment]
Over to @Coenego for review.

**Comment by bp323** [comment]
Reassigning to myself to fix the merge conflict and bring it up to date with the scss work that has gone in.

**Comment by bp323** [comment]
Over to @Coenego for review.

File: shared/gh/mobile/index.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            <div id="gh-right-container">
+                <div id="gh-header-container">
+                    <div id="gh-header"><!-- --></div>
+                </div>
+                <main id="gh-main"><!-- --></main>
+            </div>
+            <div class="clearfix"></div>
+        </div>
+
+        <footer></footer>
```
Comment (Coenego) [inline shared/gh/mobile/index.html]: Just a formality, but can you add `<!-- -->` inside the footer tags?

**Comment by Coenego** [comment]
Looks good! Only one minor comment left. Back to @bp323

**Comment by bp323** [comment]
Back to @Coenego for review.

**Pull request merged**